surah/src/commonMain/kotlin/com/islamversity/surah/SurahState.kt

```diff
@@ -20,7 +21,14 @@ data class SurahState(
                 startFrom = 0,
                 showBismillah = false,
                 bismillah = "",
-                closeScreen = false
+                closeScreen = false,
+                scrollToAya = null
             )
     }
-}
\ No newline at end of file
+}
+
+data class ScrollToAya(
+    val id: String,
+    val order: Long,
+    val position: Int,
```

Comment:
The `position` parameter just is used, what's the use case of `id, order`? (even `ScrollToAya`)

Reply:
Not sure really
But I was thinking just in case other platforms need the id of the row instead of position